client/src/components/FormField/RegisteredMFAMethodListField/AccountResetUI.js

```diff
@@ -28,7 +28,26 @@ class AccountResetUI extends Component {
    * Sends a reset request to the provided endpoint, and updates the component's state based on
    * the contents of the response.
    */
-  handleSendReset() {
+  async handleSendReset() {
```

Comment:
Do we need any kind of polyfill to support async methods?

Reply:
AFAICT it's supported by babel given our browser target configuration.